opensrp-reveal/src/main/java/org/smartregister/reveal/task/JavaHttpServerTask.java

```diff
@@ -0,0 +1,34 @@
+package org.smartregister.reveal.task;
+
+import android.content.Context;
+import android.os.AsyncTask;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.server.JavaHTTPServer;
+
+import static org.smartregister.reveal.util.Constants.DG_ID_PLACEHOLDER;
+
+/**
+ * Created by Richard Kareko on 2/4/20.
+ */
+
+public class JavaHttpServerTask extends AsyncTask<Void, Void, Void> {
+    Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)